test_unstructured_ingest/test-ingest-azure-cognitive-search.sh

```diff
@@ -4,11 +4,13 @@ set -e
 
 SCRIPT_DIR=$(dirname "$(realpath "$0")")
 cd "$SCRIPT_DIR"/.. || exit 1
-OUTPUT_FOLDER_NAME=s3
+OUTPUT_FOLDER_NAME=s3-azure-dest
```

Comment:
```suggestion
OUTPUT_FOLDER_NAME=s3-azure_cog_search-dest
```
?

...since we have a few different azure products